pkg/operator/controller.go

```diff
@@ -346,12 +383,14 @@ func (c *Controller) syncTunedDefault() (*tunedv1.Tuned, error) {
 	}
 
 	// Tuned resource found, check whether we need to update it
-	if reflect.DeepEqual(crMf.Spec, cr.Spec) {
+	if reflect.DeepEqual(crMf.Spec.Profile, cr.Spec.Profile) &&
+		reflect.DeepEqual(crMf.Spec.Recommend, cr.Spec.Recommend) {
 		klog.V(2).Infof("syncTunedDefault(): Tuned %s doesn't need updating", crMf.Name)
 		return cr, nil
 	}
 	cr = cr.DeepCopy() // never update the objects from cache
-	cr.Spec = crMf.Spec
+	cr.Spec.Profile = crMf.Spec.Profile
```

Comment:
Are we specifying the fields so we don't copy the `meta`?

Reply:
No. You don't want to overwrite the `ManagementState` that might have been changed by the admin.